NineChronicles.Headless/GraphTypes/ActionQuery.cs

```diff
@@ -320,6 +320,33 @@ public ActionQuery(StandaloneContext standaloneContext)
                     return Codec.Encode(action.PlainValue);
                 }
             );
+            Field<NonNullGraphType<ByteStringType>>(
+                "prepareRewardAssets",
+                arguments: new QueryArguments(
+                    new QueryArgument<NonNullGraphType<AddressType>>
+                    {
+                        Name = "rewardPoolAddress",
+                        Description = "address of reward pool for charge reward."
+                    },
+                    new QueryArgument<NonNullGraphType<ListGraphType<FungibleAssetValueInputType>>>
```

Comment:
Does it can also include `null` in the `assets` list?

Reply:
no, i'll fix it.